core/sql/exp/ExpLOBaccess.cpp

```diff
@@ -2280,8 +2280,10 @@ Ex_Lob_Error ExLobsOper (
 	  return LOB_GLOB_PTR_ERROR;
 	}
     }
-
-  if ((globPtr != NULL) && (operation != Lob_Init))
+  if (globPtr != NULL)
+  {
+    lobGlobals = (ExLobGlobals *)globPtr;
+    if ((operation != Lob_Init) && (operation != Lob_Cleanup))
```

Comment:
It doesn't hurt anything (and an optimizing compiler will optimize it out) but it looks like line 2288 duplicates line 2285.